packages/language-server/lib/language-service-manager.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * @typedef {import('typescript')} ts
+ * @typedef {import('typescript').IScriptSnapshot} IScriptSnapshot
+ * @typedef {import('typescript').LanguageService} LanguageService
+ */
+
+import { fileURLToPath, pathToFileURL } from 'node:url'
+
+import { createMDXLanguageService } from '@mdx-js/language-service'
+
+import { getDocByFileName } from './documents.js'
+
+/**
+ * @param {ts} ts
+ * @returns {(fileName: string) => IScriptSnapshot | undefined} XXX
```

Comment:
Is this a placeholder for future documentation? Or is there an issue with the return?